utils/souper2llvm.in

```diff
@@ -379,13 +379,13 @@ def rewriteOps(inst, newregs, ops):
 def printFuncHeader():
     # global declarations: dummy store, intrinsics
     print "target datalayout = \"e-m:e-i64:64-f80:128-n8:16:32:64-S128\"\n"
-    print "@dummy1 = global i32 0, align 4"
-    print "@dummy2 = global i32 0, align 4"
-    print "@dummy3 = global i32 0, align 4"
-    print
+    #print "@dummy1 = global i32 0, align 4"
+    #print "@dummy2 = global i32 0, align 4"
+    #print "@dummy3 = global i32 0, align 4"
+    #print
```

Comment:
This is required for `printFuncFooter()` (that is not being used anymore), so we should probably delete it.